### DISCUSSION TITLE
Windows installer support for ARM possible?

### DISCUSSION THREAD

**Discussion by jtree-spx** [espanso/espanso#1180] [Feature requests and Ideas]
Hi - installer completes but fails to execute on Surface Pro X:

Setup dialog error shown when choosing to start Espanso at the end of install.

Unable to execute file: <path to espanso.exe>

CreateProcess failed; code 216.
This version of %1 is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.

Thanks

**Comment by federico-terzi** [comment]
Hey 
Thanks for your interest! I'll do some research and let you know once I find something.
Cheers :)

**Comment by AucaCoyan** [comment]
Hi! A long time has passed since you asked the first time. Sorry for the delay!
We are growing as a community now, and I think we don't have the capacity to add another architecture, let alone bugfix everything that doesn't work in windows ARM 😢 .
Sorry!

**Reply by AucaCoyan** [reply]
A new issue was created to track this epic
https://github.com/espanso/espanso/issues/2306
